### ISSUE TITLE
Got stuck after invalid choice in "Attack for Handshake"

### ISSUE THREAD

**Issue by patrickteivas** [v1s1t0r1sh3r3/airgeddon#295]
#### What is your airgeddon version?

9.21

#### What is your Linux O.S. and which version?

Kali Linux RaspberryPi 2019.2a 32bit

#### Which is the chipset of your wireless card?

Ralink Technology, Corp. RT2070/RT3070

#### Describe the issue and the steps to reproduce it

Hm, I think I found a bug, steps to reproduce:
* Put network card to monitor mode
* Select `Handshake tools menu` [5]
* Select `Explore for targets` [4]
* Select target (Only you are allowed to test)
* Select `Capture Handshake` [5]

Now, in `Attack for Handshake` menu insert invalid choice (`-` for me). Now you are not able to select `Return top Handshake tools menu` [0] and get
```
sleep: missing operand
Try 'sleep --help' for more information.
```
Also reproducible after a failed attack for a handshake. Steps
From same menu (`Attack for Handshake`)
*  Select `Deauth aireplay attack` [2]

Now we need to **NOT** capture handshake and get `It seems we failed` message. Insert invalid choice (`-` for me). Now you are not able to select `Return Handshake tools menu` [0], **but** now will not get the error message. 
P.S. You can't select only `Return Handshake tools menu`, other choices are selectable.

**Comment by OscarAkaElvis** [comment]
Got it. Reproduced.

![image](https://user-images.githubusercontent.com/5803348/64074903-a752a600-ccb1-11e9-82fe-bd017421d8b6.png)


Thanks for the report. We'll check it soon

**Comment by v1s1t0r1sh3r3** [comment]
This is already fixed. It will be released on next v9.22 .

Anyway, you can check it on dev branch doing `git clone -b dev https://github.com/v1s1t0r1sh3r3/airgeddon.git`

If you can check if it is ok for you (it is for me on my tests), could be nice. Thanks.

**Comment by patrickteivas** [comment]
Working for me.
